source/Weapon.cpp

```diff
@@ -201,7 +201,7 @@ void Weapon::drawDecal(const Point3& point, const Vector3& normal, bool hit) {
 
 void Weapon::clearDecals() {
 	while (m_currentMissDecals.size() > 0) {
-		m_scene->remove(m_currentMissDecals.pop());
+		m_currentMissDecals.pop();
```

Comment:
This fixes a long standing bug where there'd be a crash occasionally when switching sessions or experiments.